cis/encryption.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import os
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+from cis import settings
+
+
+kms = boto3.client('kms')
+
+
+def encrypt(plaintext):
```

Comment:
May want to consider adding encryption context as optional arg.  In case we want to scope a specific set of attributes to specific grants.